README.md

```diff
@@ -1,11 +1,74 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/ngrok/ngrok-go.svg)](https://pkg.go.dev/github.com/ngrok/ngrok-go)
```

Comment:
This link also needs to be updated from `github.com/ngrok/ngrok-go` to https://pkg.go.dev/golang.ngrok.com/ngrok